src/main/java/org/folio/innreach/domain/service/impl/CirculationServiceImpl.java

```diff
@@ -499,6 +508,48 @@ private void updateTransactionHold(TransactionHold existingTransactionHold, Tran
     BeanUtils.copyProperties(pickupLocation, existingTransactionHold.getPickupLocation(), PICKUP_LOC_IGNORE_PROPS_ON_COPY);
   }
 
+  private void reportVerifyPatron(InnReachTransaction transaction) {
+    var payload = new HashMap<>();
+    var patronHold =  (TransactionPatronHold) transaction.getHold();
+    var visiblePatronId = patronHold.getPatronId();
+    var patronAgencyCode = patronHold.getPatronAgencyCode();
+    var patronName = patronHold.getPatronName();
+    payload.put("visiblePatronId", visiblePatronId);
+    payload.put("patronAgencyCode", patronAgencyCode);
+    payload.put("patronName", patronName);
+    callD2irCircOperation(D2IR_VERIFY_PATRON, transaction, payload);
+  }
+
+  private void callD2irCircOperation(String operation, InnReachTransaction transaction, Map<Object, Object> payload) {
+    var centralCode = transaction.getCentralServerCode();
+    var trackingId = transaction.getTrackingId();
+    var requestPath = resolveD2irCircPath(operation, trackingId, centralCode);
+    try {
+      if (payload == null) {
+        innReachExternalService.postInnReachApi(centralCode, requestPath);
+      } else {
+        innReachExternalService.postInnReachApi(centralCode, requestPath, payload);
+      }
+    } catch (InnReachException e) {
+      log.warn("Unexpected D2IR response: {}", e.getMessage(), e);
+    }
+  }
+
+  private String normalizePatronName(String patronFullName) {
+    String[] patronName = patronFullName.split(",");
+    String firstNameOrPreferredFirstName = null;
+    String lastName = null;
+    for (String word : patronName) {
+      String[] checkForName = word.trim().split(" ");
+      if (checkForName.length >= 2) {
+        firstNameOrPreferredFirstName = word.trim();
+      } else {
+        lastName = word.trim() + ", ";
+      }
+    }
+    return lastName + firstNameOrPreferredFirstName;
+  }
```

Comment:
You should be able to reuse logic of `PatronInfoService` for getting central patron name/type values